docs/Containers/Container-Tasks.md

```diff
@@ -0,0 +1,279 @@
+via the host.create command, as this feature is designed to run entire
+tasks within containers, rather than spinning them up within the
+existing model.
+
+This feature is also distinct from the existing distro-based Docker
+pools. These were created with the aim of running containers within our
+existing host pool by having containers operate the Evergreen agent
+inside our EC2 hosts that we initiated. However, this solution was slow
+due to the necessity of the container distro rebuilding its image every
+time the agent was released, which led to increased latency. It also
+lacked flexibility of image choice, and each host could only run a
+maximum of one container at a time, and the same container was also used
+across tasks, leading to reusability issues.
+
+This offering aims to be an improvement that addresses these issues by
+leveraging the advantages of native containerization, enabling better
+speed, flexibility, and efficiency for running tasks.
+
+## What's Different
+
+While tasks running on containers come with the same general
+capabilities you're familiar with in regular host tasks, ultimately they
+are not hosts themselves, meaning certain low-level system processes
+such as systemd present on the host that may not be readily available
+within a container. To learn more, the [docker security
+documentation](https://docs.docker.com/engine/security/) explains the
+features of Docker's isolation model. There are a couple of differences
+to note as we roll out this new feature:
+
+1.  Task Groups: The initial release of container tasks does not support
+configuring task groups. This is something we aim to support in
+future iterations.
+
+2.  Priority: The priority setting feature will not be available during
+the initial release for container tasks.
+
+3.  Greater configurability: When running container tasks, resources
+such as CPU and memory usage must be explicitly configured, unlike
+in the current distro model. Users are also responsible for
+picking the image to use, and all required software is downloaded
+during runtime. Furthermore, the container your task runs on is
+dedicated solely to that task and is never reused, so you are free
+to do whatever you want with it without needing to worry about
+leaving the environment in a messy state for the next task.
+
+# How to get started
+## Image lifecycle and self service
+
+At the moment, our image security automation is still [under
+development](https://docs.google.com/document/d/1MMePuL5YBjJQcNdtwzU2kMLPSsRLzDyE0rhTVkmXDqo/edit#heading=h.ghiwibz2opmd),
+so for the time being, modifying a project\'s respective Dockerfile
+requires the submission of a PR against our repo.
+
+1.  Stage a PR in our centralized image
+    [repository](https://github.com/evergreen-ci/container-initial-offering-dockerfiles).
+    This repo hosts Dockerfiles for all teams participating in the
+    beta of running container tasks in Evergreen. The PR, when merged
```

Comment:
May be worth transitioning the "beta" wording to more like "experimental" since the beta offering is done.